docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -47,9 +47,11 @@ cluster.initial_master_nodes:
   - master-c
 --------------------------------------------------
 
-You can use a mix of IP addresses and node names too. If there is more than one
-Elasticsearch node with the same IP address then the transport port must also
-be given to specify exactly which node is meant:
+You can also use IP addresses, or even a mix of IP addresses and node names. If
+there is more than one {es} node with the same IP address then the IP addresses
+can be distinguished by appending each node's transport port as shown below.
+However, {es} nodes should have unique node names so {es} does not allow two
+nodes with the same name to be distinguished by their transport port like this.
```

Comment:
I find this last sentence here very confusing, with what's presented as a logical conclusion (... so {es} does not allow ...). The problem we're trying to address is that users  think  that this setting is resolved in the usual way (i.e. same as `discovery.seed_hosts`), but we require exact matching on this one, in particular `host-name:port` is not an acceptable combination.

I wonder if we should just say that the values of this setting are not resolved and need to exactly match either the node's name (which defaults to the host name), or the publish address, which can be presented as an IP address or an IP:PORT pair. We can even add a sentence to the effect that it is generally preferable to identify nodes uniquely through their node names.

Reply:
Ok, I had clearly spent too long staring at this, and now I don't like this wording either. This setting is a pain to document. I have tried again in 61b5f6c.